lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -96,7 +96,7 @@ def test_start_flow_not_verified(self, course_mode):
             PayAndVerifyView.PHOTO_ID_REQ,
             PayAndVerifyView.WEBCAM_REQ,
         ])
-        self._assert_upgrade_session_flag(False)
+        self._assert_upgrade_session_flag(None)
```

Comment:
I think we still want this set to False to protect against the case in which someone starts the upgrade flow but doesn't complete payment.